worker/raft/manifold.go

```diff
@@ -99,22 +99,30 @@ func (config ManifoldConfig) start(context dependency.Context) (worker.Worker, e
 }
 
 func raftOutput(in worker.Worker, out interface{}) error {
-	w, ok := in.(withRaft)
+	w, ok := in.(withRaftOutputs)
 	if !ok {
-		return errors.Errorf("expected input of type %T, got %T", w, in)
+		return errors.Errorf("expected input of type *Worker, got %T", in)
```

Comment:
I guess the fact that %T was used (and is used elsewhere) indicates that we care about informing about the expected type. It is really just for developers right? Once it works, it will never be seen by the end user, but it does say more accurately what's amiss.

Reply:
I'll change it to `withRaftOutputs`. Amusingly, the %T just reports `<nil>` here when you get the error, not the expected type  - I saw it when I hadn't added `LogStore()` to the mock.